tests/post_training/pipelines/base.py

```diff
@@ -92,11 +158,9 @@ def get_result_dict(self):
             "Metric value": self.metric_value,
             "Metric diff": self.metric_diff,
             "Num FQ": self.num_fq_nodes,
-            "RAM MiB": self.format_memory_usage(self.quant_memory_usage),
-            "Quant. time": self.format_time(self.time_quantization),
-            "Stat. collection time": self.time_stat_collection,
-            "Bias correction time": self.time_bias_correction,
-            "Validation time": self.time_validation,
+            "RAM MiB": self.format_memory_usage(self.compression_memory_usage),
+            "Compression time": self.format_time(self.time_compression),
```

Comment:
Rename it to `Compr. time`. Looks to long for report.

Reply:
Done